### PR TITLE
Update contributions LP footer

### PR DESCRIPTION
**Pull request by TomPretty**
## What are you doing in this PR?
Update the footer on the LP page to make more compliant! It also stylistically bring it more inline with the rest of the support site. [Here's the trello link](https://trello.com/c/cu3TkHQG/2453-landing-page-legal-requirements-footer)

### mobile
Screenshot 2020-12-03 at 11 33 43

### tablet 
Screenshot 2020-12-03 at 11 50 25

### desktop
Screenshot 2020-12-03 at 11 50 43

### wide
Screenshot 2020-12-03 at 11 35 09

### CODE REVIEW

File: support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { gaEvent } from 'helpers/tracking/googleTagManager';
 import * as storage from 'helpers/storage';
 import { set as setCookie } from 'helpers/cookie';
 import Page from 'components/page/page';
-import Footer from 'components/footer/footer';
```
Comment (tomrf1) [inline support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx:21]: is this component still used elsewhere?

**Review by tomrf1 — APPROVED** [review]
looks great!

File: support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx (hunk 1 above)
Comment (TomPretty) [reply support-frontend/assets/pages/contributions-landing/contributionsLanding.jsx:21]: looks like just the paypal error page, and the TY page

File: support-frontend/assets/components/footerCompliant/ContributionsFooter.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      <div css={alignmentStyles}>
+        <Footer faqsLink={faqsLink} termsConditionsLink={termsConditionsLink}>
+          <p>
+            The ultimate owner of the Guardian is The Scott Trust Limited, whose
+            role it is to secure the editorial and financial independence of the
+            Guardian in perpetuity. Reader contributions support the Guardian’s
+            journalism. Please note that your support of the Guardian’s
+            journalism does not constitute a charitable donation, as such your
+            contribution is not eligible for Gift Aid in the UK nor a
+            tax-deduction elsewhere.
```
Comment (paulbrown1982) [inline support-frontend/assets/components/footerCompliant/ContributionsFooter.jsx:50]: The *Contact us* has moved into the footer.

**Pull request merged**